src/components/UserPortal/ChatRoom/ChatRoom.spec.tsx

```
@@ -5505,169 +5314,42 @@ const MARK_CHAT_MESSAGES_AS_READ_MOCK = [
-      },
-    },
-    result: {
-      data: {
-        markChatMessagesAsRead: {
-          _id: '1',
-        },
-      },
-    },
-  },
+  })),
 ];
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)